app/views/activities/poll.js.erb

```diff
@@ -91,4 +91,4 @@
 <% end %>
 
 ActivitiesPoller.lastPoll = <%= @this_poll %>;
-ActivitiesPoller.poll();
+<%# ActivitiesPoller.poll(); %>
```

Comment:
was it a mistake to leave this commented out?

Reply:
yes it is. reverting...